ui/src/components/Tasks/DailyTasks/index.tsx

```diff
@@ -64,13 +64,14 @@ const selectMoreOptions = [
 ];
 
 // const NoTodaysCommitment: React.FC<{ addItemCb: () => void }> = (p) => (
-const NoTodaysCommitment = () => (
+const NoTodaysCommitment: React.FC<{ openSlidebar: () => void }> = (p) => (
   <div className={c('not-found-container')}>
     <NoItemsFound
       Icon={RiPlayListAddLine}
       message=" You haven't added a status update for today."
       addItemText="Add one"
       addItemUrl=""
+      addItemCb={() => p.openSlidebar()}
```

Comment:
Since no arguments are needed for the called function, we can simply do:
```suggestion
      addItemCb={p.openSlidebar}
```